### PR TITLE
Minor GUI fixes

### LINKED ISSUE #491: [BUG] Can't zoom in all the way
You use to be able to zoom in on large imagery by an undefined amount, now you can't seem to zoom in more than the image width, there's some hard constraint stopping you from zooming further

### LINKED ISSUE #492: [BUG] Desktop - open buttons on main menu different sizes but only when window small
spacing

### LINKED ISSUE #496: [FEATURE] Desktop - minor title issues on mainpage
On the main desktop page I would say DIVE Annotation Tool not VIAME Annotation Tool (though keep VIAME on top left), just to reduce duplication of saying VIAME everywhere and because short-term (at least foreseeable 3 months) there's going to be multiple annotation tools packaged in VIAME unless we get feature completeness in just one

### PR DESCRIPTION
**Pull request by subdavis**
fixes #491
fixes #492
fixes #496

### CODE REVIEW

File: client/src/components/annotators/useMediaController.ts [hunk 1]
```diff
@@ -71,7 +71,12 @@ export default function useMediaController({ emit }: {
       right: params.map.maxBounds.right,
       bottom: params.map.maxBounds.bottom,
     });
-    geoViewerRef.value.zoomRange(params.map);
+    geoViewerRef.value.zoomRange({
+      // do not set a "zoom out" limit so that 1x(map size) is always min.
+      min: -Infinity,
```
Comment (subdavis) [inline client/src/components/annotators/useMediaController.ts:76]: This seems to have the desired effect: always make min zoom the size of the map.

I can't figure out _why_ it does this, but I figured @BryonLewis will know.

**Comment by BryonLewis** [comment]
Closing because this was mainly for ease of demonstration.

**Comment by subdavis** [comment]
This is my PR, not yours :P

**Comment by subdavis** [comment]
Fixed.  I had a dumb math problem.  zero identity is hard.

File: client/platform/desktop/components/Recent.vue [hunk 2]
```diff
@@ -36,12 +36,15 @@ export default defineComponent({
...
       <v-col>
         <v-row>
-          <v-col cols="8">
+          <v-col
+            md="8"
+            sm="6"
+          >
             <h1 class="text-h3 mb-4 font-weight-light">
-              VIAME Annotation Tool
+              Dive Annotation Tool
```
Comment (BryonLewis) [inline client/platform/desktop/components/Recent.vue:44]: Terribly sorry, should this be 'DIVE' now?

**Review by BryonLewis — CHANGES_REQUESTED** [review]
Just the DIVE question.

Noting this in case we ever have to come back to this:
Just noting that the caveat with 4x zoom level is that the most you will get for zooming in pixel resizing is (dimension*2^4) so a 256x256 image would become 4096x4096  which should be sufficient in most cases (I don't think anyone out there has an 8K monitor/TV they are using this on).  This is separate from window resizing though, that can cause you to have a zoom level outside of the clamped min/max levels.  I don't think we need to adjust this now, but just noting for eventual searching if we have to address this again.

**Comment by subdavis** [comment]
Changed it in the electron build config as well.

**Review by BryonLewis — APPROVED** [review]
Looks good

**Pull request merged**